app/models/collection.rb

```diff
@@ -152,6 +154,7 @@ def entity_types
       projects: 'Projects',
       reports: 'Reports',
       cohorts: 'Cohorts',
+      supplemental_data_sets: 'Supplemental Data Sources',
```

Comment:
I think everywhere else you are calling it `Supplemental Data Sets` is there a reason this is different?